x-pack/plugins/canvas/common/lib/escape_reg_exp.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function escapeRegExp(string) {
+  return string.replace(/[.*+?^${}()|[\]\\\/]/g, '\\$&');
```

Comment:
This changes the escaping slightly, escaping `/` when it wasn't before. And this method is used in `queryEsSQL`, so it'll escape a new string in SQL queries.

I *think* forward slash may be an illegal character for ES indices and fields though, so it might be a non-issue. But I'd feel better about the change hearing from someone with more ES knowledge.

Reply:
Idk if this applies only to index names, but `/` is in their list of illegal characters here: https://www.elastic.co/guide/en/elasticsearch//reference/current/indices-create-index.html